src/Scripting/ScriptingSystem.cpp

```diff
@@ -64,33 +64,37 @@ void ScriptingSystem::_initPackageTable() {
     // Usage in Lua:
     // local gameBindings = require "bindings.game" -- If the module starts with 'bindings.' we try to load/create the binding table.
     // gameBindings.doSomething()
-    _solState->add_package_loader([this](const std::string &module) {
+    //
+    // Note that sol::as_function is required here because some compilers mangle (mis-mangle?) different lambdas
+    // inside the same function using the same name, and this leads to problems.
```

Comment:
I would say "having the same signature" instead of "using the same name" as they don't have a name.

Reply:
Fixed